desktop/ui/src/main/java/org/datacleaner/windows/OptionsDialog.java

```diff
@@ -89,8 +94,8 @@
 
     @Inject
     protected OptionsDialog(WindowContext windowContext, DataCleanerConfiguration configuration,
-                            UserPreferences userPreferences, DatabaseDriversPanel databaseDriversPanel,
-                            ExtensionPackagesPanel extensionPackagesPanel) {
+            UserPreferences userPreferences, DatabaseDriversPanel databaseDriversPanel,
+            ExtensionPackagesPanel extensionPackagesPanel) {
```

Comment:
Another codestyle difference. Didn't we get the Czech team to use the defaults files?

We really need to keep an eye out for these differences so we can harmonize codestyles.